testing/test_modules.sh

```diff
@@ -36,5 +40,9 @@ cat $TEST_LIST | while read module args; do
     fi
 done
 
+echo TAPTAP
```

Comment:
TAPTAP?

Reply:
That was some local debugging... removed now!